lib/systems/physics_system.cpp

```diff
@@ -18,28 +18,28 @@ void PhysicsSystem::update(double deltatime) {
         }
 
         auto transform = entityManager->getComponent<TransformComponent>(entityId);
+        // TODO make it so trigger goes through everything instead of only checking if player can move through trigger
```

Comment:
This is fixed when we do the AABB thing?

Reply:
Yes.